openreview/conference/builder.py

```diff
@@ -1499,6 +1491,17 @@ def post_decision_stage(self, reveal_all_authors=False, reveal_authors_accepted=
         def is_release_authors(is_note_accepted):
             return reveal_all_authors or (reveal_authors_accepted and is_note_accepted)
 
+        def decision_to_venue(decision_option):
```

Comment:
could you move this function to the tools module? I think we are going to need it to update the `content.venue` for previous conferences.